src/ipa_pointcloud_to_laserscan_nodelet.cpp

```diff
@@ -149,7 +149,7 @@ void IpaPointCloudToLaserScanNodelet::cloudCb(const sensor_msgs::PointCloud2Cons
     }
     catch (tf2::TransformException ex)
     {
-      NODELET_WARN_STREAM("Transform failure: " << ex.what());
+      NODELET_DEBUG_STREAM("Transform failure: " << ex.what());
```

Comment:
I'd vote for investigating/fixing the `lookupTransform` rather than just suppressing the output...